kubernetes/cray-console-operator/Chart.yaml

```diff
@@ -44,5 +44,5 @@ annotations:
     - name: cray-console-operator
       image: artifactory.algol60.net/csm-docker/S-T-A-B-L-E/cray-console-operator:0.0.0-image
     - name: alpine
-      image: alpine:latest
+      image: artifactory.algol60.net/csm-docker/stable/docker.io/library/alpine:latest
```

Comment:
Shouldn't this be `.../apline:3`, like the other change?

Reply:
Good catch - don't know how that slipped through - thanks!